README.asciidoc

```diff
@@ -0,0 +1,274 @@
+     _         _   ___      _   
+  _ | |__ _ __| |_| _ ) ___| |_ 
+ | || / _` / _| / | _ \/ _ \  _|
+  \__/\__,_\__|_\_|___/\___/\__|
+                                
+### Team members: 
+
+              Karl Villers - MPE
+
+              Karl Rahn - ME
+              
+              Matis Vahter - P
+              
+              Nikita Kurenkov - PE
+              
+### FUSION FILES:
+              
+https://gmail1315991.autodesk360.com/g/projects/20220905552790856/data/dXJuOmFkc2sud2lwcHJvZDpmcy5mb2xkZXI6Y28uMTlSNUpCd09UQ1NRQ2FLUjZ3LUhLQQ==

```

Comment:
I'll remove the gmail link as it needs an account, although with the other link which I added displays the full robot and probably when downloading it, it will download the whole project.